cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -43,44 +44,48 @@
-  private static final String EXCLUDED_TOPICS_PARAM = "excluded_topics";
-  private static final String USER_TASK_IDS_PARAM = "user_task_ids";
-  private static final String SKIP_URP_DEMOTION_PARAM = "skip_urp_demotion";
-  private static final String EXCLUDE_FOLLOWER_DEMOTION_PARAM = "exclude_follower_demotion";
-  private static final String DISABLE_SELF_HEALING_FOR_PARAM = "disable_self_healing_for";
-  private static final String ENABLE_SELF_HEALING_FOR_PARAM = "enable_self_healing_for";
-  private static final String EXCLUDE_RECENTLY_DEMOTED_BROKERS_PARAM = "exclude_recently_demoted_brokers";
+  public static final String JSON_PARAM = "json";
+  public static final String START_MS_PARAM = "start";
+  public static final String END_MS_PARAM = "end";
+  public static final String ENTRIES_PARAM = "entries";
+  public static final String ALLOW_CAPACITY_ESTIMATION_PARAM = "allow_capacity_estimation";
+  public static final String CLEAR_METRICS_PARAM = "clearmetrics";
+  public static final String TIME_PARAM = "time";
+  public static final String VERBOSE_PARAM = "verbose";
+  public static final String SUPER_VERBOSE_PARAM = "super_verbose";
+  public static final String RESOURCE_PARAM = "resource";
+  public static final String REASON = "reason";
+  public static final String DATA_FROM_PARAM = "data_from";
+  public static final String KAFKA_ASSIGNER_MODE_PARAM = "kafka_assigner";
+  public static final String MAX_LOAD_PARAM = "max_load";
+  public static final String GOALS_PARAM = "goals";
+  public static final String BROKER_ID_PARAM = "brokerid";
+  public static final String TOPIC_PARAM = "topic";
+  public static final String PARTITION_PARAM = "partition";
+  public static final String DRY_RUN_PARAM = "dryrun";
+  public static final String THROTTLE_ADDED_BROKER_PARAM = "throttle_added_broker";
+  public static final String THROTTLE_REMOVED_BROKER_PARAM = "throttle_removed_broker";
+  public static final String IGNORE_PROPOSAL_CACHE_PARAM = "ignore_proposal_cache";
+  public static final String USE_READY_DEFAULT_GOALS_PARAM = "use_ready_default_goals";
+  public static final String CONCURRENT_PARTITION_MOVEMENTS_PER_BROKER_PARAM = "concurrent_partition_movements_per_broker";
+  public static final String CONCURRENT_LEADER_MOVEMENTS_PARAM = "concurrent_leader_movements";
+  public static final String DEFAULT_PARTITION_LOAD_RESOURCE = "disk";
+  public static final String SUBSTATES_PARAM = "substates";
+  public static final String MIN_VALID_PARTITION_RATIO_PARAM = "min_valid_partition_ratio";
+  public static final String SKIP_HARD_GOAL_CHECK_PARAM = "skip_hard_goal_check";
+  public static final String EXCLUDED_TOPICS_PARAM = "excluded_topics";
+  public static final String USER_TASK_IDS_PARAM = "user_task_ids";
+  public static final String CLIENT_IDS_PARAM = "client_ids";
+  public static final String ENDPOINTS_PARAM = "endpoints";
+  public static final String TYPES_PARAM = "types";
```

Comment:
Looks like `Set<String> userTasks = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);` (line 201) below is missing `CLIENT_IDS_PARAM`, `ENDPOINTS_PARAM`, `TYPES_PARAM`, and `ENTRIES_PARAM`.
Hence, wouldn't it cause the user receive an exception if they try to use any of the above parameters with `user_tasks` endpoint?
-- Maybe I am missing something?